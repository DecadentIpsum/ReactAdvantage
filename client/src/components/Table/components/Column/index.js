import React, { Component } from 'react';
import { Column as PrimeColumn } from 'primereact/components/column/Column';

export default class Column extends Component {
    render() {
        return (
            <PrimeColumn {...this.props}>
                {this.props.children}
            </PrimeColumn>
        );
    }
}
