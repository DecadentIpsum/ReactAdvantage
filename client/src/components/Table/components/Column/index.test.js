import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import Column from './index';

it('renders correctly', () => {
    const component = shallow(<Column />);

    expect(component).toMatchSnapshot();
});