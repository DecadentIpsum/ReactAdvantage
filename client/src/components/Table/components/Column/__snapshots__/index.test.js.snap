// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Column />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "body": null,
      "children": undefined,
      "className": null,
      "colSpan": null,
      "columnKey": null,
      "editor": null,
      "editorValidator": null,
      "expander": false,
      "field": null,
      "filter": false,
      "filterElement": null,
      "filterFunction": null,
      "filterMatchMode": "startsWith",
      "filterMaxLength": null,
      "filterPlaceholder": null,
      "filterType": "text",
      "footer": null,
      "frozen": false,
      "header": null,
      "rowReorder": false,
      "rowReorderIcon": "fa fa-bars",
      "rowSpan": null,
      "selectionMode": null,
      "sortField": null,
      "sortFunction": null,
      "sortable": false,
      "style": null,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "body": null,
        "children": undefined,
        "className": null,
        "colSpan": null,
        "columnKey": null,
        "editor": null,
        "editorValidator": null,
        "expander": false,
        "field": null,
        "filter": false,
        "filterElement": null,
        "filterFunction": null,
        "filterMatchMode": "startsWith",
        "filterMaxLength": null,
        "filterPlaceholder": null,
        "filterType": "text",
        "footer": null,
        "frozen": false,
        "header": null,
        "rowReorder": false,
        "rowReorderIcon": "fa fa-bars",
        "rowSpan": null,
        "selectionMode": null,
        "sortField": null,
        "sortFunction": null,
        "sortable": false,
        "style": null,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
